examples/planning/README.md

```diff
@@ -0,0 +1,57 @@
+# Introduction to Planning for Self Driving Vehicles
+Planning is an essential piece of the Autonomous Vehicles (AV) stack.
+
+In this series of notebooks you will train and evaluate a data-driven ML policy.
+
+**Before starting, please download the [Lyft L5 Prediction Dataset 2020](https://self-driving.lyft.com/level5/prediction/) and follow [the instructions](https://github.com/lyft/l5kit#download-the-datasets) to correctly organise it.**
+
+Training an effective ML policy for planning can takes several hours on the most performing hardware.
+For this reason, we provide trained models you can experiment with in our evaluations notebooks,
```

Comment:
I don't see instructions how to use them in the open- and closed-loop evaluation notebooks.